aeneas/src/vst/Verifier.v3

```diff
@@ -886,29 +893,61 @@ class VstCompoundVerifier {
 						_ => ;
 					}
 				}
-				if (rep.length > bitWidth) errAtToken(token).BitPatternTooLong(bitWidth, rep.length);
-			}
-			Tuple(list) => {
-				for (l = list.list; l != null; l = l.tail) {
-					checkPackingExpr(env, l.head, MAX_PACKING_WIDTH); // TODO: proper size
-				}
-			}
-			App(p, args) => { // TODO
-				// TODO: resolve and apply packing {p}
-				for (l = args.list; l != null; l = l.tail) {
-					checkPackingExpr(env, l.head, MAX_PACKING_WIDTH); // TODO: proper size
+				return byte.!(rep.length);
+			}
+			Concat(l) => {
+				var total: byte = 0;
+				for (e = l.list; e != null; e = e.tail) {
+					var s = inferPackingExpr(env, e.head);
+					if (s == MAX_PACKING_WIDTH || total == MAX_PACKING_WIDTH) total = MAX_PACKING_WIDTH;
```

Comment:
IIUC this should be checking for the `total` doesn't exceed `MAX_PACKING_WIDTH`,  but it is only checking for equality, and will miss the case where the last expression goes over the limit.